dayrepo/snippets/urls.py

```diff
@@ -14,5 +14,6 @@
     path("checklist/", views.checklist_post,name="checklist_post"),
     path("edit/<int:snippet_id>/", views.snippet_edit,name="snippet_edit"),
     path("edit/checklist/<int:checklist_id>/", views.checklist_edit,name="checklist_edit"),
+    path("delete/<int:target_id>/<str:delete_type>/", views.db_delete,name="post_delete"),
```

Comment:
post  って命名は残す？メソッドに合わせる？
```suggestion
    path("delete/<int:target_id>/<str:delete_type>/", views.db_delete,name="db_delete"),
```

Reply:
対応しました
https://github.com/bob-g12/dayrepo-for-transportation/pull/56/commits/9b7423f4332aa675cccfde71a8615cb1d5dc3e8d